dbt/seeds/ccao/schema.yml

```diff
@@ -0,0 +1,35 @@
+seeds:
+  - name: ccao.class_dict
+    description: '{{ doc("seed_class_dict") }}'
+    config:
+      column_types:
+        min_age: double
+        max_age: double
+        min_size: double
+        max_size: double
```

Comment:
Make sure these types align with the descriptions from the R file (they should probably be integers).

Reply:
I do not believe `integer` can handle `Infinity` in presto - all integer types have [max values](https://prestodb.io/docs/current/language/types.html#integer). I also just can't get 'Infinity' to cast as an int, but it will cast as a double in Athena.